apps/ui/package.json

```diff
@@ -59,7 +62,6 @@
     "@tanstack/vue-query": "^5.64.2",
     "@vueuse/core": "^10.4.1",
     "@walletconnect/core": "^2.11.0",
-    "@walletconnect/ethereum-provider": "2.10.6",
     "@walletconnect/modal": "2.6.2",
```

Comment:
this should all be removed

Reply:
We use some of those (`core`, `utils`, `types`) to implement our WalletConnect wallet for building executions, but `modal` can be removed.